iconoir.com/components/Footer.tsx

```diff
@@ -13,29 +32,81 @@ export function Footer() {
         </LogoIcon>
         <Logo src={'/iconoir-logo.svg'} alt={'Iconoir Logo'} />
       </LogoContainer>
-      <FooterNavigationItem
-        as={'a'}
-        href={LICENSE_LINK}
-        target={'_blank'}
-        rel={'noreferrer'}
-      >
-        License
-      </FooterNavigationItem>
+      <FooterCategories>
+        <FooterCategory category={'Project'} links={[
+          {name: 'Our Mission',url:'/support'},
+          {name: 'Contribute',url:'/docs/contributing'},
+          {name: 'Donate',url:'https://opencollective.com/iconoir/donate'}
+        ]} />
+        <FooterCategory category={'Support'} links={[
+          {name: 'License',url:'https://github.com/iconoir-icons/iconoir/blob/main/LICENSE'},
+          {name: 'GitHub Repository',url:'https://github.com/iconoir-icons/iconoir'},
+          {name: 'File a Request',url:'https://github.com/iconoir-icons/iconoir/issues/new/choose'}
+        ]} />
+        <FooterCategory category={'Developers'} links={[
+          {name: 'Changelog',url:'/docs/changelog'},
+          {name: 'React and React Native',url:'/docs/libraries/iconoir-react'},
+          {name: 'Flutter',url:'/docs/libraries/flutter'},
+          {name: 'Framer and Figma',url:'/docs/libraries/framer'},
+          {name: 'CSS',url:'/docs/libraries/css'}
+        ]} />
+        
+        
+      </FooterCategories>
+      <FooterEnd>
+        <Text13 style={{ fontWeight: 400 }}>Parts of this content are ©2020-2023 by individual Iconoir contributors. Content available under a <a href={'https://github.com/iconoir-icons/iconoir/blob/main/LICENSE'} target={'_blank'} rel={'noreferrer'}>MIT License</a>.</Text13>
+        <Text13 style={{ fontWeight: 400 }}><a href={'https://www.freeprivacypolicy.com/live/ba00d743-a0cd-44f8-8cb5-6f58911db0fb'} target={'_blank'} rel={'noreferrer'}>Privacy</a></Text13>
```

Comment:
We should maybe copy this to the repository as Markdown rather than linking to the third-party. And then once it's in the repository you can probably create a new documentation item to lead to it [here.](https://github.com/iconoir-icons/iconoir/blob/main/iconoir.com/pages/docs/%5B...slug%5D.tsx#L104)

That, or I would just link it to the Markdown on GitHub for now and we can fuss with having it render properly on the website later.

Reply:
Agree. I chose the more pragmatic solution. Noted for later 👍